sdl_android/src/main/java/com/smartdevicelink/proxy/interfaces/ISdl.java

```diff
@@ -149,6 +150,20 @@
      */
     boolean removeOnRPCNotificationListener(FunctionID notificationId, OnRPCNotificationListener listener);
 
+    /**
+     * Add an OnRPCResponseListener for specified response
+     * @param responseId FunctionID of the response that is to be listened for
+     * @param listener listener that should be added for the response ID
+     */
+    void addOnRPCResponseListener(FunctionID responseId, OnRPCResponseListener listener);
```

Comment:
This doesn't make sense. The response listener is specific to a correlation id sent by the request. This should only be addOnRPCListener and take a OnRPCListener. The remove method should be modified accordingly.